tests/test_schema.py

```diff
@@ -0,0 +1,113 @@
+from typing import Literal
+
+from draive import DataModel, Field
+
+
+class SchemaNestedModel(DataModel):
+    value: Literal["A", "B", "C"] = Field(description="selection")
+    other_value: None
+
+
+class SchemaModel(DataModel):
+    str_value: str
+    int_value: int
+    float_value: float
+    bool_value: bool
+    list_value: list[str]
+    optional_value: str | None = Field(description="alternative")
+    nested: SchemaNestedModel = Field(alias="nested_value", description="alternative")
+
```

Comment:
Add field of type `dict`. I'm not sure if it will be converted.

Reply:
it won't as for now - this is why I have skipped it 😉